site2/docs/reference-pulsar-admin.md

```diff
@@ -2383,6 +2386,30 @@ Usage
 $ pulsar-admin topics remove-deduplication tenant/namespace/topic
 ```
 
+### `set-max-subscriptions`
+Set the maximum number of subscriptions for a topic.
+
+Usage
+```bash
+$ pulsar-admin topics set-max-subscriptions tenant/namespace/topic options
```

Comment:
It looks good, but I have some questions:
What does `tenant/namespace/topic` mean?

If we use this command,a topic name should be passed in here, such as:
persistent://tenant/namespace/topic

The entire command looks like this:
pulsar-admin topics set-max-subscriptions [topic name] [options]